README.md

```diff
@@ -379,7 +379,7 @@ Use the `./api-regression -config <filename>` option to specify your configurati
 # api-regression configuration
 #
 # Properties:
-#   'ip' is required and is the IP Address of the storage controller used for testing
+#   'mc-ip-addrs:' is required and is the IP Addresses of the storage controller(s) used for testing. At least one address must be specified
```

Comment:
It would be good to mention that this is an ordered pair, with communications established at first to the first member of the pair.